content/modeling/modeling_3.md

```diff
@@ -14,56 +14,97 @@ The snowplow_fractribution dbt package comes with a list of variables specified
 
-There are five path transform options available:
-- Exposure (default in this package): the same events in succession are reduced to one: `A → A → B` becomes `A → B`. A compromise between first and unique.
-- UniquePath: all events in a path are treated as unique (no reduction of complexity). Best for smaller datasets (small lookback window) without a lot of retargeting.
-- First: keep only the first occurrence of any event: `A → B → A` becomes `A → B`. Best for brand awareness marketing.
-- Frequency: keep a count of the events' frequency: `A → A → B` becomes `A(2) → B`. Best when there is a lot of retargeting.
+There are six path transform options available:
+- `exposure_path` (default in this package): the same events in succession are reduced to one: `A → A → B` becomes `A → B`. A compromise between first and unique.
+- `unique_path`: all events in a path are treated as unique (no reduction of complexity). Best for smaller datasets (small lookback window) without a lot of retargeting.
+- `first_path`: keep only the first occurrence of any event: `A → B → A` becomes `A → B`. Best for brand awareness marketing.
+- `frequency_path`: keep a count of the events' frequency: `A → A → B` becomes `A(2) → B`. Best when there is a lot of retargeting.
+- `remove_if_last_and_not_all`: requires a channel to be added as a parameter, which gets removed from the latest paths unless it removes the whole path as it is trying to reach a non-matching channel parameter: E.g target element: A path: `A → B → A → A` becomes `A → B`.
+- `remove_if_not_all`: requires a channel to be added as a parameter, which gets removed from the path altogether unless it would result in the whole path's removal: E.g target element: A path: `A → B → A → A` becomes `B​`.
 
 
 #### **Step 2:** Configure macros
 
 **Configure the conversion_clause macro**
 
-The conversion_macro specifies how to filter Snowplow events to only conversion events. How this is filtered will depend on your definition of a conversion. The default is filtering to events where `tr_total > 0`, but this could instead filter on `event_name = 'checkout'`, for example. If you are using the e-commerce model, you will still need to set this for the fractribution code to run (even though all events are conversions in the e-commerce model), just change it to `transaction_revenue > 0`.
+The conversion_macro specifies how to filter Snowplow events to only conversion events. How this is filtered will depend on your definition of a conversion. The default is filtering to events where `tr_total > 0`, but this could instead filter on `event_name = 'checkout'`, for example. If you are using the e-commerce model, you will still need to set this to enable the models to run (even though all events are conversions in the e-commerce model), just change it to `transaction_revenue > 0`.
```

Comment:
This copy may need to be changed to reflect that it defaults to the ecomm package now rather than `tr_total`.

Reply:
The sample data still has `tr_total` as it is simpler to prepare and it is done before we talk about using the ecommerce package in the tracking steps but I added a sentence to clarify this! Also added a third option for the `conversions_source` where I describe that they don't have to do anything extra if they used the sample data just define that as the conversion source in their project.yml.